src/components/Foods.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import foodSeed from '../foods.json'
+import FoodBox from './FoodBox'
+import AddFood from './AddFood'
+import TodayFood from './TodayFood'
+
+
+class Foods extends React.Component {
+    constructor(props){
+        super(props)
+        this.state ={
+            foodList: foodSeed,
+            filteredFoods: foodSeed,
+            formShow: false,
+            TodayFoods: []
+        }
+    }
+
+    formAppear = () => {
+        this.setState({
+            formShow: true
```

Comment:
if instead of setting the formShow to be true you change it into the opposite of the current value, you could add the feature to toggle it on and off. Instead of having it appear only once.